pilot/lib/pilot/tester_manager.rb

```diff
@@ -43,6 +40,27 @@ def add_tester(options)
       end
     end
 
+    def add_tester_to_groups(tester: nil, app: nil)
+      default_external_group = app.default_external_group
+      if default_external_group.nil? && config[:groups].nil?
+        UI.error("The app #{app.name} does not have a default external group.")
+        UI.error("Please make sure to pass group names to the `:groups` option.")
+        return
```

Comment:
Also, this commit indicates that there is no longer a concept of default external groups for new applications since the recent Testflight update; unless I misunderstood.

https://github.com/fastlane/fastlane/pull/8943/files/bf03c6463a38763bf1c4a940b24c14375636a5ae#r112680533